pytorch_blade/bazel_build.py

```diff
@@ -118,6 +117,21 @@ def __init__(self, *args, **kwargs):
         if running_on_ci():
             self.configs += ["--config=ci_build"]
 
+        # ----------------------------------------------------------------------- #
+        # --------------------   Settings for Quantization   -------------------- #
+        # ----------------------------------------------------------------------- #
+        # This is for quantization process starts from blade_compression.
+        # blade_compression uses fx graph to do quantization, which is
+        # introduced in torch 1.8.0. So the lower bound of torch version is
+        # set to 1.8.0
+        # note: If we use backend (such as trt) to do calibration and get the
+        # quantization engine, this limitation should not be set.
+        is_enable_quantization = self.torch_major_version == 1 and \
+                                    8 <= self.torch_minor_version < 12
+        if is_enable_quantization:
+            self.torch_extra_opts.append("--copt=-DTORCH_BLADE_BUILD_QUANTIZATION")
```

Comment:
Nit: Please move this `--copt` to the `build:torch_enable_quantization` line in `.bazelrc`

Reply:
done